config/jobs/kubernetes-sigs/node-feature-discovery/OWNERS

```diff
@@ -0,0 +1,5 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- kad
+- marquiz
```

Comment:
So they will get assigned for review by blunderbuss
```suggestion
approvers:
- kad
- marquiz
reviewers:
- kad
- marquiz
```

Reply:
OK, fixed